Build.PL

```diff
@@ -90,7 +90,7 @@ EOW
       "recursive_test_files" => 1,
       "requires" => {
         "Carp" => 0,
-        "DBD::SQLite" => "1.64",
+        "DBD::SQLite" => "1.68",
```

Comment:
You don't need to include changes to Build.PL or META.json as these files are generated.